### DISCUSSION TITLE
(Minor?) problem with validity of booster vaccination?

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#198] [Q&A] [answered]
If someone was vaccinated with Johnson and Johnson (sequence/total numbers = 1/1) and gets his
first booster vaccination with Biontech, Moderna or AstraZeneca vaccine (sequence/total numbers = 2/2) then this certicate is not valid until 2 weeks later.

This is somehow confusing...

Reason should be: This certificate can not be distinguished from a normal second dose with Biontech, Moderna or AstraZeneca. 

Workaround: Usage of older certificate 1/1 until new certificate is valid...

Only a minor problem?

See also: https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/Implementation.md

**Comment by Ein-Tim** [comment]
There is no possibility to fix this issue, as, how you said, there is no option to differentiate between a normal "2/2" certificate with e.g. Biontech and a "2/2" certificate with e.g. Biontech after one already received one shot of J&J. 

One can use the workaround you gave, no idea if this is recommended, but it definitely works 😅

**Comment by mwicke** [comment, marked as answer]
@mge-medisoftware that is correct, we are aware of this issue with the schema. And yes, a workaround can be to use the 1/1 certificate in the meantime.

**Reply by mge-medisoftware** [reply]
Ok, thank you all, we will add an appropriate hint for the doctor/medical staff. May be reasonable for the Corona-Warn-App too

**Reply by Ein-Tim** [reply]
@mwicke 

You are aware of this issue means you will fix it? I thought this isn't possible?

**Comment by mdebono** [comment]
The [Guidelines on Technical Specifications for EU Digital COVID Certificates JSON Schema Specification Schema version 1.3.0 2021-06-09](https://ec.europa.eu/health/sites/default/files/ehealth/docs/covid-certificate_json_specification_en.pdf) quoted in https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/Implementation.md, refers to boosters only as being 3/3 and says nothing about a booster after Johnson & Johnson being 2/2.

This mismatch between certificates and apps will be causing confusion at verification points such as allowing entrance to several places implementing 2G+ with reference to boosters, e.g. [Baden-Württemburg (German)](https://sozialministerium.baden-wuerttemberg.de/de/service/presse/pressemitteilung/pid/ausnahmen-von-der-testpflicht-bei-2g-plus/), and potentially other countries that will have restrictions referring to booster shots.

In light of this, the proposed workaround seems to be the best solution for (J&J booster) certificates that have already been issued, and in general the mismatch gap should be closed.

**Comment by Ein-Tim** [comment]
FYI: https://ec.europa.eu/commission/presscorner/detail/en/ip_21_6837 says:

"Boosters will be recorded as follows:

3/3 for a booster dose following a primary 2-dose vaccination series.
2/1 for a booster dose following a single-dose vaccination or a one dose of a 2-dose vaccine administered to a recovered person."
